server/resp/src/main/java/org/infinispan/server/resp/commands/string/SETRANGE.java

```diff
@@ -1,16 +1,18 @@
 package org.infinispan.server.resp.commands.string;
 
-import io.netty.buffer.Unpooled;
-import io.netty.channel.ChannelHandlerContext;
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.commons.util.Util;
 import org.infinispan.server.resp.Consumers;
 import org.infinispan.server.resp.Resp3Handler;
 import org.infinispan.server.resp.RespCommand;
 import org.infinispan.server.resp.RespRequestHandler;
 import org.infinispan.server.resp.commands.ArgumentUtils;
 import org.infinispan.server.resp.commands.Resp3Command;
 
-import java.util.List;
-import java.util.concurrent.CompletionStage;
+import io.netty.buffer.Unpooled;
```

Comment:
This Unpooled here is a bit worrying. I guess the reviewer in the original change wasn't very sharp :grin:. We should update to use the allocator.